src/Compilers/CSharp/Test/Symbol/DocumentationComments/CrefTests.cs

```diff
@@ -6871,5 +6871,80 @@ record CacheContext(string String)" + terminator;
             var symbol = model.GetSymbolInfo(crefSyntaxes.Single()).Symbol;
             Assert.Equal(SymbolKind.Property, symbol.Kind);
         }
+
+        [Fact]
+        public void ExtensionInheritedMembersInSemanticModelLookup()
+        {
+            var source = """
+explicit extension Base for object
+{
+    int P => throw null;
+}
+
+explicit extension Derived for object : Base
+{
+}
+
+/// <see cref='Derived.P'/>
+class C
+{
+}
+""";
+            var comp = (Compilation)CreateCompilation(source, targetFramework: TargetFramework.Net70,
+                parseOptions: TestOptions.RegularPreview.WithDocumentationMode(DocumentationMode.Diagnose));
+
+            // Not expected to bind, since we don't consider inherited members.
```

Comment:
>since we don't consider inherited members [](http://example.com/codeflow?start=37&length=41)

You mean specifically in doc comments or in general?

Reply:
In cref. I'm following similar rules to that of interfaces, although I don't think those are spec'ed...